ui/raidboss/data/04-sb/raid/o6s.txt

```diff
@@ -93,11 +93,11 @@ hideall "--sync--"
 590 "Release" sync / 1[56]:[^:]*:Demon Chadarnook:2804:/
 
 603 "Possession" sync / 1[56]:[^:]*:Demon Chadarnook:2803:/ window 20
-611 "Easterlies" sync / 03:........:Easterly:/ 
+611 "Easterlies" sync / 03:........:Easterly:/
 613 "Rock Hard" sync / 1[56]:[^:]*:Portrayal of Earth:2812:/
 614 "Demonic Wave" sync / 1[56]:[^:]*:Portrayal of Water:2830:/
 622 "Demonic Howl" sync / 1[56]:[^:]*:Demon Chadarnook:282B:/
 626 "Featherlance" sync / 1[56]:[^:]*:Easterly:283E:/
 
-626 "--sync--" sync / 1[56]:[^:]*:284D:Demon Chadarnook:/
+626 "--sync--" sync / 1[56]:[^:]*:Demon Chadarnook:284D:/
```

Comment:
This looks suspiciously like a `StartsUsing` sync that was mislabeled. I'm fairly sure I screwed this up during the Endwalker conversion. https://github.com/quisquous/cactbot/blame/beb375702898ee4da6d4e7ef4320d080949fef81/ui/raidboss/data/04-sb/raid/o6s.txt#L101 from right before that conversion has that with the very old `StartsUsing` syntax.

Reply:
Oooooh good catch, fixed.